netcore/Makefile

```diff
@@ -74,6 +72,17 @@ all: $(addprefix build-, $(TEST_SUITES))
 
 check: $(addprefix run-, $(TEST_SUITES))
 
-clean:
-	rm -rf sdk shared host dotnet tests LICENSE.txt ThirdPartyNotices.txt $(NETCORESDK_FILE)
+FEED_BASE_URL = https://dotnetfeed.blob.core.windows.net/dotnet-core
+TEST_ASSETS_URL = https://dotnetfeed.blob.core.windows.net/dotnet-core/corefx-tests/4.6.0-preview4.19156.10/OSX.x64/netcoreapp/corefx-test-assets.xml
```

Comment:
s/4.6.0-preview4.19156.10/$(VERSION)/ ?

Reply:
Oh, nvm, it uses different prefix for the version. It's hard to keep track of it.

Reply:
No idea what this version is and how it relates to netcore version.

Reply:
I actually do have an idea, but this comment space is not big enough to write the description of the Maestro (buildtools) / Maestro++ (arcade) dependency management and how the SDK picks the correct assets. It's quite confusing and I keep bumping into it.

Reply:
https://github.com/dotnet/corefx/blob/b5ac532676652673c4e72fcc99e734309a034f56/eng/Versions.props#L6-L10

Reply:
Looks like there's something strange I don't get. Apparently the current assets are still 4.6.0 while I would expect them to be 4.7.0. Unfortunately the builds are now published only on the feeds and MyGet is no longer updated [properly] so it's difficult to figure out what is going on. I guess it's fine to hard code the version for now and figure this out later.

Reply:
@ViktorHofer could you suggest a way to keep them in sync manually?

Reply:
Sorry I'm currently OOF till Monday next week. As the PR is merged it seems you were successful?

Reply:
We are downloading some version of the test assets, just not sure how to pick the correct version.

Reply:
Either with a DARC subscription to corefx or with mirror-updates from coreclr. cc @jkotas

Reply:
I think you should use DARC subscription.

You want to be trailing corefx master, for both the CoreFX binaries and CoreFX tests. DARC subscription will auto-submit PR whenever there is a new good official CoreFX build to pick up.

>  Apparently the current assets are still 4.6.0 while I would expect them to be 4.7.0.

There are way too many different version numbers in CoreFX that you need a secret decoder ring to understand. The build version number is still 4.6.X for these.

Reply:
A Darc subscription is definitely preferable but I'm not sure how much this would require this repository to be arcade enabled. @wtgodbe @safern @cchosta should know.

> a secret decoder ring to understand

We tried to align versions but we still have some differences. We should do a better job here in the future. cc @ericstj

Reply:
> A Darc subscription is definitely preferable but I'm not sure how much this would require this repository to be arcade enabled. @wtgodbe @safern @cchosta should know.

In order to setup a Darc subscription should be pretty simple. We just need to add a`eng` folder if it doesn't exist and throw a couple of files there, `Version.Details.xml` and `Versions.props` -- for example here are the ones in coreclr who uses this channel: 

https://github.com/dotnet/coreclr/blob/master/eng/Version.Details.xml#L14
https://github.com/dotnet/coreclr/blob/master/eng/Versions.props#L11

Setting up the subscription in maestro is fairly simple, just a matter of adding it to the database -- @jcagme can give more details on that.

Reply:
We have a ton of constraints on assembly versioning due to netstandard, oobing, desktop support, etc, etc.  It's difficult to wiggle much in this space but we could make an effort to rebase all the assembly versions for netcoreapp.

The specific 4.6 / 4.7 discrepancy is due to the switch to arcade mid cycle which reset the file-version algorithm.  At the time folks chose to bump the file-versions to keep them monotonically increasing while keeping the packages versioned at 4.6.  We could choose to fix that now.  The major.minor of the packages is arbitrary and meant to be 1:1 with a release wave.

Reply:
@filipnavara is the version discrepancy currently causing pain? If yes could you please describe the scenarios.

@vargaz we need to create the subscription as shortly explained by @safern above and prepare the repo for it. I think it makes sense to create an issue for it where we can discuss this further.

Reply:
>  is the version discrepancy currently causing pain? If yes could you please describe the scenarios.

Not in technical way. It is just confusing when trying to figure out what version should I look for and when manually verifying that my setup is correct (happens too often now without the DARC subscription).